src/Server.php

```diff
@@ -18,14 +18,26 @@ public function __construct(LoopInterface $loop)
 
     public function listen($port, $host = '127.0.0.1')
     {
-        if (strpos($host, ':') !== false) {
+        $localSocket = '';
+        if (filter_var($host, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
```

Comment:
@cboden f.x. PCRE (for preg_replace) is like Filter enabled per default to.

Is it maybe better to preg_match both instead of using  filter?